SystemTests/roles/robot/files/k8s_methods.robot

```diff
@@ -63,7 +63,7 @@ Post Request With Json Body
 
 Submit SparkPi Job
     [Arguments]   ${job_name}
-    ${submitbody}=    Create Dictionary   name=${job_name}   language=Scala   main_class=org.apache.spark.examples.SparkPi    path_to_main_app_file=local:///opt/spark/examples/jars/spark-examples_2.11-2.4.0.jar
+    ${submitbody}=    Create Dictionary   name=${job_name}   language=Scala   main_class=org.apache.spark.examples.SparkPi    path_to_main_app_file=local:///opt/spark/examples/jars/spark-examples_2.11-2.4.0.jar      arguments=["10"]
```

Comment:
We should keep one system test without any arguments passed in: this will make it easier to establish whether it's the supplying of arguments that is causing any future problems.